docs/docusaurus.config.js

```diff
@@ -88,11 +88,10 @@ const config = {
         },
       },
       navbar: {
-        title: 'IMMICH',
```

Comment:
Why did you remove the title? Did you mean to use an immich with the text in it? Right now it no longer appears and I think it should be added back.

Reply:
Yeah, I can add the logo with text. The logo with text is vectorized, we don't have the license to use Router Medium directly as a font on the web, but its vectorized form and images is fine